docs: add file type support warning to migration guide (#9295)
vercel/ai: content/docs/08-migration-guides/26-migration-guide-5-0.mdx

}
```

<Note>
  Some models do not support text files (text/plain, text/markdown, text/csv,

>   etc.) as file parts. For text files, we recommend to read the content and send as a text part

Reviewer: ```suggestion
  etc.) as file parts. For text files, you can read and send the context as a text part
```

  instead:

```tsx
// Instead of this:
{ type: 'file', data: buffer, mediaType: 'text/plain' }

// Do this:
{ type: 'text', text: buffer.toString('utf-8') }
```

> 

Reviewer: ```suggestion
```
